srcpkgs/parprouted/files/parprouted/conf

```diff
@@ -0,0 +1,2 @@
+#EXIT_IF=wlan0
+#IN_IF=eth0
```

Comment:
We usually don't ship conf files if we would do that then we would also have to mark them as configuration file.